### PR TITLE
Js/hc 1556 add model version

### PR DESCRIPTION
**Pull request by Glumli**

### CODE REVIEW

File: src/services/fhirService.js [hunk 1]
```diff
@@ -98,6 +98,7 @@ const fhirService = {
                     encrypted_tags: results[1],
                     date: dateUtils.formatDateYyyyMmDd(record.customCreationDate || new Date()),
                     attachment_key: record.attachmentKey,
+                    data_model_version: DATA_MODEL_VERSION,
```
[the remark this answers is not in the document]
Comment (Glumli) [reply src/services/fhirService.js]: Uh wow. Your right. Just checked iOS

File: package.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "hc-sdk-js",
   "version": "0.1.0",
```
Comment (ibihim) [inline package.json]: We should start to use the version field within package.json

File: test/services/fhirServiceTest.js [hunk 3]
```diff
@@ -139,6 +139,7 @@ describe('services/fhirService', () => {
         }).default;
 
         fhirServiceUploadRecordSpy = sinon.spy(fhirService, 'uploadRecord');
+        global.DATA_MODEL_VERSION = 1;
```
Comment (ibihim) [inline test/services/fhirServiceTest.js]: Where do you verify, that the DATA_MODEL_VERSION matches what you set here?

**Comment by Glumli** [comment]
@ibihim PTAL

**Pull request merged**